04-Docker.md

```diff
@@ -1,62 +1,76 @@
 # Part IV: Docker
 
-## Objectives: 
-By the end of this module, the SRE will: 
-- Describe the directives from a sample dockerfile 
-- Explain layers and the docker cache 
-- Start a docker container that runs ubuntu linux 
+## Objectives:
+By the end of this module, the SRE will:
+- Describe the directives from a sample dockerfile
+- Explain layers and the docker cache
+- Start a docker container that runs ubuntu linux
 - Describe the following container concepts:
   - base image
   - single pid
-  - command vs entrypoint
+  - `CMD` vs `ENTRYPOINT`
   - logs
   - volumes
   - layers and ordering
   - caching
   - build
   - push
 
-## Deliverables 
-1. Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
+## Deliverables
+Complete the project described below. Answer the questions, then schedule a time to meet with your mentor and discuss the what you've built.  
 
-### Docker 
+### Docker
+  1. Setup
+  - https://www.youtube.com/watch?v=JprTjTViaEA
+  - https://jonnylangefeld.github.io/learning/Docker/How%2Bto%2BDocker.html
   - Install docker
-  - `docker login` to reactiveops quay and dockerhub
+  - login to reactiveops quay and dockerhub via the terminal
+    - You will need to be given access to these, reach out to your pod leader.
```

Comment:
Pod leader is specific to FWOPs, I think I'd prefer to keep the language more generic.

Reply:
Done!